service/internal/cloudconfig/template/aws-cni.go

```diff
@@ -101,7 +101,7 @@ spec:
       tolerations:
         - operator: Exists
       containers:
-        - image: {{.RegistryDomain}}/giantswarm/amazon-k8s-cni:v1.6.0
+        - image: {{.RegistryDomain}}/giantswarm/amazon-k8s-cni:v{{.AWSCNIVersion}}
```

Comment:
Can we safely assume the `v` prefix madness will not hunt us down at some point? I am really not certain how we define it in all the different places.

Reply:
all component versions have just the sematic numbers, so having one with `v` in release would look weird